README.md

```diff
@@ -674,7 +674,8 @@ lora_target_modules:
 #  - gate_proj
 #  - down_proj
 #  - up_proj
-lora_target_linear: # If true, will target all linear layers
+lora_target_linear: # If true, will target all linear modules
+peft_layers_to_transform: # The layer indices to transform, otherwise, apply to all layers
```

Comment:
setting the prefix for this to `peft_` since we are eventually going to migrate the other `lora_` options soon.

Reply:
What do you think of using `peft_override` or `peft_config` dictionary which accepts the same keys as in the Peft config?

This would reduce the need for us to keep updating for each change.

The change would follow the same structure as `cfg.model_config`